views/components/Sidebar/SidebarFooter.test.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { MemoryRouter } from 'react-router-dom'
+import { render, getByText, getByAltText } from '@testing-library/react'
```

Comment:
let's use `screen`. That way we can remove the `container` constants as well.

```suggestion
import { render, screen } from '@testing-library/react'
```

Reply:
[465c369](https://github.com/AMP-SCZ/dpdash/pull/458/commits/465c36952e0f8176957c65d89dfea1c6b86ca296)

Reply:
@Elkrival this hasn't been resolved